src/main/java/de/dennisguse/opentracks/services/TrackDeleteService.java

```diff
@@ -137,35 +122,28 @@ public TrackDeleteService getService() {
     }
 
     public static class DeleteStatus {
-        private final int progress;
-        private final int max;
-        private final Track.Id trackId;
+        private final List<Track.Id> trackIds;
 
         /**
-         * @param trackId  Track.Id just deleted if any.
-         * @param progress number of deletes.
-         * @param max      total of deletes to be done.
+         * @param trackIds List of deleted Track.Ids.
          */
-        private DeleteStatus(@Nullable Track.Id trackId, int progress, int max) {
-            this.trackId = trackId;
-            this.progress = progress;
-            this.max = max;
+        private DeleteStatus(@Nullable List<Track.Id> trackIds) {
+            this.trackIds = trackIds;
         }
 
         public boolean isFinished() {
-            return progress == max;
```

Comment:
This is not needed anymore?

Reply:
Actually the whole `isFinished()` could be removed, because the TrackDeletionService only emmits the `DeleteStatus` once at the end. So we can remove it completely.

Reply:
I've renamed the `DeleteStatus` class to `DeletionFinishedStatus` to make this clear.